packages/server-wallet/src/wallet/multi-threaded-wallet/worker.ts

```diff
@@ -25,7 +33,11 @@ async function startWorker() {
   const logger = createLogger(walletConfig);
 
   logger.debug(`Worker %o starting`, threadId);
-  const wallet = await Wallet.create(walletConfig);
+  const wallet = (await Wallet.create(walletConfig)) as SingleThreadedWallet;
```

Comment:
Could we just call `SingleThreadedWallet.create` instead? That way we avoid the cast.

Reply:
c56aade0bd3c9fe39640d5fb10e42d4ff5469f30